spec/metric_fu/metrics/reek/generator_spec.rb

```diff
@@ -109,11 +102,6 @@
 
       it "should NOT insert nil smells into the array when there's a newline in the method call" do
         expect(@matches.last[:code_smells]).to eq(@matches.last[:code_smells].compact)
-        expect(@matches.last).to eq(file_path: "app/controllers/newline_controller.rb",
```

Comment:
This test is no longer relevant since the new line fix from above means this smell is no longer generated. If you look you will see that this smell is the second half of this message:

  >   [6, 9]:NewlineController#some_method calls current_user.<< "new line\n" multiple times (Duplication)

Note the embedded new line....